### DISCUSSION TITLE
Ignorable path parts / shadowed folder names

### DISCUSSION THREAD

**Discussion by Mahfoudh94** [withastro/roadmap#1022] [Proposal]
# Summary

Influenced by routing in Next.js, we need a way to group elements like having a page in `src/pages/(markets)/home.astro` corresponds to `https://example.com/home` (the `(market)` in the path was ignored), this would allow for easier grouping, especially for content generation

# Background & Motivation

The main source of the idea is next.js and some routing solutions for front-end frameworks
I felt the need for it in the content generation for SSG sites, optimized for speed and load time

**Comment by austinm911** [comment]
duplicate of https://github.com/withastro/roadmap/discussions/480, you should go ahead and upvote that one as well
